File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1094.c
#include <stdio.h>

int main(){
    int n, v, s=0, r=0, c=0, i;
    char animal;
    scanf("%d", &n);
    for(i=0;i<n;i++){
        scanf("%d %c", &v, &animal);
        if(animal=='C'){
            c+=v;
        } else if(animal=='R'){
            r+=v;
        } else{
            s+=v;
        }
    }
    double soma = c+r+s;

    printf("Total: %d cobaias\n", c+r+s);
    printf("Total de coelhos: %d\n", c);
    printf("Total de ratos: %d\n", r);
    printf("Total de sapos: %d\n", s);
    printf("Percentual de coelhos: %.2f \%\n", (100*c)/soma);
    printf("Percentual de ratos: %.2f \%\n", (100*r)/soma);
    printf("Percentual de sapos: %.2f \%\n", (100*s)/soma);

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1094.rb
n = gets.to_i
s=0
r=0
c=0

for i in 1..n do
    ns = gets.chomp.split(' ')
    v = ns[0].to_i
    animal = ns[1]
    if(animal=='C') then
        c+=v;
    elsif(animal=='R') then
        r+=v;
    else
        s+=v;
    end
end
soma = (c+r+s).to_f

puts "Total: #{c+r+s} cobaias"
puts "Total de coelhos: #{c}"
puts "Total de ratos: #{r}"
puts "Total de sapos: #{s}"
puts "Percentual de coelhos: %0.02f %%" % ((100*c)/soma).round(2)
puts "Percentual de ratos: %0.02f %%" % ((100*r)/soma).round(2)
puts "Percentual de sapos: %0.02f %%" % ((100*s)/soma).round(2)
    